### PR TITLE
Remove unused deps

### LINKED ISSUE #2490: Knowing about dependencies and dev dependencies
Hi
First of all I want to thank you to all of the contributors for this repository. I am regularly following this repo for near about 4-5 months, now some questions arise in my mind.

Please tell me the requirement of adding (dev dependencies)
1. ```@types/tapable```
2. ```@types/webpack```
3. ```@types/vfile-message```
4. ```babel-core```
5. ```cross-spawn```

I searched the whole project and couldn't get any details regarding above packages.

I thought maybe the type definitions for webpack is needed to get better intellisense(VSCode) in webpack.config.js files, but for me it is not working, and hopefully it is the expected behavior.

It is not an issue, just a quick discussion(maybe convert into issue later if any). If you want me to ask this doubts somewhere else, please ping me the website😅

### PR DESCRIPTION
**Pull request by jooohhn**
- Removed deps that don't seem to be used in the project
- Fixes #2490

### CODE REVIEW

File: configs/webpack.config.renderer.dev.babel.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ import fs from 'fs';
 import webpack from 'webpack';
 import chalk from 'chalk';
 import merge from 'webpack-merge';
-import { spawn, execSync } from 'child_process';
+import { spawn, execSync } from 'cross-spawn';
```
Comment (amilajack) [inline configs/webpack.config.renderer.dev.babel.js:13]: Why this change? Not sure if we’re using this dep

Comment (jooohhn) [reply configs/webpack.config.renderer.dev.babel.js:13]: It was already in the `package.json` and I thought it might have been needed. 

Is it not the case?

Comment (amilajack) [reply configs/webpack.config.renderer.dev.babel.js:13]: Might have been accidentally added. It’s not used anywhere

Comment (sdc224) [reply configs/webpack.config.renderer.dev.babel.js:13]: Actually we don't need `cross-spawn` for executing process, `child_process` is enough
